docs/content/preview/explore/observability/active-session-history.md

```diff
@@ -31,28 +31,27 @@ To use ASH, enable and configure the following flags for each node of your clust
 
 | Flag | Description |
 | :--- | :---------- |
-| allowed_preview_flags_csv | Pass the flags `ysql_yb_ash_enable_infra` and `ysql_yb_enable_ash` in this flag in CSV format. |
+| allowed_preview_flags_csv | Set the value of this flag to include `ysql_yb_ash_enable_infra` and `ysql_yb_enable_ash`. |
```

Comment:
```suggestion
| allowed_preview_flags_csv | Set the value of this flag to include `ysql_yb_ash_enable_infra`,`ysql_yb_enable_ash`. |
```

While setting the flag, we would need a comma, that's why there was a comma here